tools/autograd/derivatives.yaml

```diff
@@ -1130,13 +1130,13 @@
   result: auto_element_wise
 
 - name: native_batch_norm(Tensor input, Tensor? weight, Tensor? bias, Tensor? running_mean, Tensor? running_var, bool training, float momentum, float eps) -> (Tensor, Tensor, Tensor)
-  input, weight, bias: "grad.defined() ? native_batch_norm_backward(grad, input, weight, running_mean, running_var, result1, result2, training, eps, grad_input_mask) : std::tuple<Tensor, Tensor, Tensor>()"
-  result0: batch_norm_jvp(input_p, input_t, weight_p, weight_t, bias_p, bias_t, running_mean, running_var, result1, result2, training, eps)
+  input, weight, bias: "grads[0].defined() ? native_batch_norm_backward(grads[0], input, weight, running_mean, running_var, result1, result2, training, eps, grad_input_mask) : std::tuple<Tensor, Tensor, Tensor>()"
+  result0, result1, result2: batch_norm_jvp(input_p, input_t, weight_p, weight_t, bias_p, bias_t, running_mean, running_var, result1, result2, training, eps)
 
 - name: native_batch_norm_backward(Tensor grad_out, Tensor input, Tensor? weight, Tensor? running_mean, Tensor? running_var, Tensor? save_mean, Tensor? save_invstd, bool train, float eps, bool[3] output_mask) -> (Tensor, Tensor, Tensor)
   input, weight, grad_out: batchnorm_double_backward(input, weight, grads[0], grads[1], grads[2], grad_out, running_mean, running_var, train, eps, save_mean, save_invstd, grad_input_mask)
-  save_mean: not_implemented("native_batch_norm_backward save_mean")
-  save_invstd: not_implemented("native_batch_norm_backward save_invstd")
+  save_mean: Tensor()
```

Comment:
Yeah they are the right gradients - I had to think for a while to realize this, but what is happening is that because save_mean and save_invstd can be both written in terms of input. batchnorm_double_backward computes the backward of batchnorm_backward as if the entire computation were computed in terms of inputs (as save_mean and save_invstd are soley used to reduce recomputation). I will update this PR to add a note for this.